src/lib/executors/Executor.ts

```diff
@@ -298,11 +298,8 @@ export default abstract class BaseExecutor<
       }
     };
 
-    const needsSeparateNotificationChain = [
-      'coverage',
-      'error',
-      'log'
-    ].includes(<string>eventName);
+    const needsSeparateNotificationChain =
```

Comment:
Instead of singling out `coverage`, `error` and `log` here and calling it `needsSeparateNotificationChain`, maybe we should name it after the actual events that are part of the test run. E.g. something like `isTestLifeCycleEvent`. Does that make sense?